test/default-downloads-test.js

```diff
@@ -129,21 +129,6 @@ describe('default-downloads', function() {
         });
       });
 
-      it('ia32 download exists', function(done) {
```

Comment:
Let's not remove a test just because it is failing. In https://github.com/vvo/selenium-standalone/pull/335 (another PR wanting to update the default versions), I indicated that if chrome is no longer shipping platform-specific version, then new logic and tests should be added in order to make sure that old versions can still be used in `selenium-standalone`.